test/mitmproxy/contentviews/test_image_parser.py

```diff
@@ -104,3 +104,50 @@ def test_parse_png(filename, metadata):
 def test_parse_gif(filename, metadata):
     with open(tutils.test_data.path(filename), 'rb') as f:
         assert metadata == image_parser.parse_gif(f.read())
+
+
+@pytest.mark.parametrize("filename, metadata", {
+    # check app0
+    "mitmproxy/data/image_parser/example.jpg": [
+        ('Format', 'JPEG (ISO 10918)'),
+        ('jfif_version', '(1, 1)'),
+        ('jfif_density', '(96, 96)'),
+        ('jfif_unit', '1'),
+        ('Size', '256 x 256 px')
+    ],
+    # check com
+    "mitmproxy/data/image_parser/comment.jpg": [
+        ('Format', 'JPEG (ISO 10918)'),
+        ('jfif_version', '(1, 1)'),
+        ('jfif_density', '(96, 96)'),
+        ('jfif_unit', '1'),
+        ('comment', "b'mitmproxy test image'"),
+        ('Size', '256 x 256 px')
+    ],
+    # check app1
+    "mitmproxy/data/image_parser/app1.jpeg": [
+        ('Format', 'JPEG (ISO 10918)'),
+        ('jfif_version', '(1, 1)'),
+        ('jfif_density', '(72, 72)'),
+        ('jfif_unit', '1'),
+        ('make', 'Canon'),
+        ('model', 'Canon PowerShot A60'),
+        ('modify_date', '2004:07:16 18:46:04'),
+        ('Size', '717 x 558 px')
+    ],
+    # check multiple segments
+    "mitmproxy/data/image_parser/image-err1.jpg": [
```

Comment:
This picture is copyrighted (http://www.gettyimages.de/license/93901920), do we have another one which could replace this test case?